src/Core/src/WindowOverlay/WindowOverlay.Android.cs

```diff
@@ -22,7 +22,11 @@ public virtual bool Initialize()
 			if (Window == null)
 				return false;
 
+			if (Window?.Content?.Handler == null)
```

Comment:
Can we roll the null check on 22 into this? And maybe create a local for Window?.Content since we access it multiple times.